src/main/java/com/somemore/recruitboard/domain/RecruitBoard.java

```diff
@@ -93,4 +102,20 @@ private void updateRecruitmentInfo(RecruitBoardUpdateRequestDto dto) {
             dto.admitted()
         );
     }
+
+    private void validateStatusChange(RecruitStatus newStatus) {
+        if (newStatus != RECRUITING && newStatus != CLOSED) {
+            throw new IllegalArgumentException("상태는 '모집중' 또는 '마감'으로만 변경할 수 있습니다.");
+        }
+    }
```

Comment:
부정 비교와 &&가 합쳐지니 가독성이 조금 떨어지는 것 같습니다.

이 부분을 개선하기 위해 상태에 대한 변경 가능 여부를 속성으로 추가하는 방식은 어떻게 생각하시나요?

``` java
public enum RecruitStatus {
    RECRUITING(true),
    CLOSED(true),
    PENDING(false),
    ARCHIVED(false);

    private final boolean changeable;

    RecruitStatus(boolean changeable) {
        this.changeable = changeable;
    }

    public boolean isChangeable() {
        return changeable;
    }
}
```

```suggestion
    private void validateStatusChange(RecruitStatus newStatus) {
        if (newStatus.isChangeable()) {
            return;
        }
        throw new IllegalArgumentException("상태는 '모집중' 또는 '마감'으로만 변경할 수 있습니다.");
    }
```

Reply:
오.. 이거 매우 좋습니다. 바로 반영하겠습니다.